src/static_functions.js

```diff
@@ -1,17 +1,25 @@
+import router from "@/router";
 
 
 export function getHeaders() {
     return {
         'Access-Control-Allow-Origin': '*',
         'Access-Control-Allow-Methods':'GET,PUT,POST,DELETE,PATCH,OPTIONS'}
 }
+
 export function getAuthenticatedHeaders () {
-    const defaultHeaders = getHeaders();
-    defaultHeaders["Authorization"] = getCookie("access_token");
    return {
        ...getHeaders(),
-       Authorization: this.$cookies('access_token')
+       Authorization: 'Bearer '+ $cookies.get('access_token')
    };
+
+
+}
+
+export function logout() {
+    $cookies.remove('access_token');
```

Comment:
you change the state here, so function is not static.
by the way you have to logout on the backend and then there will be not need to remove cookies here